pkg/geo/geomfn/rotate.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"math"
+
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/twpayne/go-geom"
+)
+
+// Rotate returns a modified Geometry whose coordinates are rotated around the origin by a rotation angle 
```

Comment:
nit: full stop at the end of these comments.
they are also above 100 chars if you don't mind new-lining them.

you'll probably run into a lint error as you have not run `crlfmt` -- you'll likely need to run `git diff --name-only -r HEAD~1 | xargs crlfmt -w -tab 2`

Reply:
I was able to pass `make lintshort`. What does the command `git diff --name-only -r HEAD~1 | xargs crlfmt -w -tab 2` do?

Reply:
just to clarify, is `make shortlint` a good check to be sure that the code also passes the lint CI pipeline?

Reply:
partially. as the name implies, it runs a subset (the fast tests).
CI runs all of them, but takes 30 min on a beefy machine. my macbook pro personally becomes a Boeing 787 when I run `make lint` :)

Reply:
Yep, I noticed that too on my Macbook for `make lint` haha! I fixed the linting issue in the recent commit.